core/utils/codel.h

```diff
@@ -135,7 +136,12 @@ class Codel final: public Queue<T> {
     return i;
   }
 
-  size_t Capacity() override { return queue_.max_size(); }
+  size_t Capacity() override { 
```

Comment:
The restriction is fine -- I think it needs to be documented. I am not super sure how one picks `max_size_` -- it seems to depend on the workload, but c'est la vie.